xlators/mount/fuse/utils/mount_glusterfs.in

```diff
@@ -549,7 +549,7 @@ EOF
         exit 1;
     }
 
-    grep_ret=$(echo ${mount_point} | grep '^\-o');
+    grep_ret=$(echo ${mount_point} | grep '^-o');
```

Comment:
nit: if you are fixing it already, you can omit the 'echo' and just grep the variable right away. Just like those 'cat ... | grep ' where the 'cat' is not needed and 'grep ...' would suffice.